### DISCUSSION TITLE
My outpainting area looks unfocused?

### DISCUSSION THREAD

**Discussion by Teedo145** [lllyasviel/Fooocus#1628] [Q&A] [answered]
![doggy](https://github.com/lllyasviel/Fooocus/assets/33029967/f1366ff0-700f-42b2-a128-b4ba0078bb25)

As you can see, the entire outpainting area that has been added is blurry? Does anyone know how to fix this or what is causing it?

**Comment by IPv6** [comment]
Make a one step in a time, not all 4 direction in the one generation

**Reply by Teedo145** [reply]
Thanks, that did improve it but still isn't quality enough to be usable :(
![image](https://github.com/lllyasviel/Fooocus/assets/33029967/06784924-481d-44a4-8ee7-816465fe5ce6)

**Reply by IPv6** [reply, marked as answer]
Blurness means resolution of the outpaint zone is too big for SDXL to handle well
You can downscale image, do outpainting (in steps) and then upscale it (via Fooocus or another AI upscaler), for example

**Reply by Teedo145** [reply]
Thanks i will give that a go. 
I'm trying to get a good quality 2048x2048 image without inpainting. I was rendering 1024x1024 and upscaling to 2048x2048 but the quality wasn't great in some areas
